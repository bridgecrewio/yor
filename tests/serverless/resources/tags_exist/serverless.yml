service: my-sls-service
provider:
  name: aws
  runtime: nodejs8.10
  region: us-east-1
  tags:
    GLOBAL_TAG1: foo
    GLOBAL_TAG2: bar
  stackTags:
    RESOURCE_TAG1: foo
    RESOURCE_TAG2: bar

functions:
  myFunction:
    handler: myFunction.handler
    tags:
      TAG1_FUNC: Func1 Tag Value
      TAG2_FUNC: Func1 Tag2 Value
      yor_trace: 5b920200-6399-4b50-8b84-73b171aca5d4
    name: myFunction
  myFunction2:
    name: myFunction2
    handler: myFunction2.handler
    tags:
      TAG1_FUNC: Func2 Tag Value
      TAG2_FUNC: Func2 Tag2 Value
      yor_trace: 258489b8-d6ab-4a28-a88d-f10ab9481e4f

resources: # CloudFormation template syntax
  Resources:
    S3BucketPublicRead:
      Type: AWS::S3::Bucket
      Properties:
        Tags:
          - Key: TAG1_CFN
            Value: CFN Tag Value
          - Key: TAG2_CFN
            Value: CFN Tag2 Value
        AccessControl: Private
        BucketEncryption:
          ServerSideEncryptionConfiguration:
            - ServerSideEncryptionByDefault:
                SSEAlgorithm: AES256