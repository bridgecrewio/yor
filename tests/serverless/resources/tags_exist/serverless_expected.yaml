service: my-sls-service
provider:
  name: aws
  runtime: nodejs8.10
  region: us-east-1
  tags:
    GLOBAL_TAG1: foo
    GLOBAL_TAG2: bar
  stackTags:
    RESOURCE_TAG1: foo
    RESOURCE_TAG2: bar

functions:
  myFunction:
    handler: myFunction.handler
    tags:
      TAG1_FUNC: Func1 Tag Value
      TAG2_FUNC: Func1 Tag2 Value
    name: myFunction
  myFunction2:
    name: myFunction2
    handler: myFunction2.handler
    tags:
      TAG1_FUNC: Func2 Tag Value
      TAG2_FUNC: Func2 Tag2 Value

resources: # CloudFormation template syntax
  Resources:
    S3BucketPublicRead:
      Type: AWS::S3::Bucket
      Properties:
        Tags:
          - Key: TAG1_CFN
            Value: CFN Tag Value
          - Key: TAG2_CFN
            Value: CFN Tag2 Value
        AccessControl: Private
        BucketEncryption:
          ServerSideEncryptionConfiguration:
            - ServerSideEncryptionByDefault:
                SSEAlgorithm: AES256
